### ISSUE TITLE
Request for Pretraining Log

### ISSUE THREAD

**Issue by whalesing** [kabouzeid/point2vec#2]
Hi, fantastic job on this! Could you provide access to the pretraining log to better understand the training process? Thanks.

**Comment by karimknaebel** [comment]
We provide the training loss curve in the README which should give you a good idea about the training process (https://github.com/kabouzeid/point2vec#point2vec-pre-training-on-shapenet).

Since it's lightning, I don't think the actual log file contains a lot of useful information, but I've attached it below anyway for your convenience :)
[Pre-training Log File](https://github.com/kabouzeid/point2vec/files/12342593/p2v-pre-sh.txt)
